### PR TITLE
fix: update to new standarts

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
* Switch to new loader script system
* Fix query

### CODE REVIEW

File: conf/IpTracker.conf.dist [hunk 1]
```diff
@@ -1,5 +1,10 @@
-[worldserver]
```
Comment (FrancescoBorzi) [inline conf/IpTracker.conf.dist:1]: `[worldserver]` should stay

**Comment by 55Honey** [comment]
I couldn't make it work. Compile and login worked fine but no data was written to the DB.

File: conf/IpTracker.conf.dist (hunk 1 above)
Comment (Winfidonarleyan) [reply conf/IpTracker.conf.dist:1]: No

**Comment by Winfidonarleyan** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/7272

**Comment by FrancescoBorzi** [comment]
> [azerothcore/azerothcore-wotlk#7272](https://github.com/azerothcore/azerothcore-wotlk/pull/7272)

how come it was working before without that?

**Comment by Winfidonarleyan** [comment]
> I couldn't make it work. Compile and login worked fine but no data was written to the DB.

Can retest?

**Comment by 55Honey** [comment]
It still does not save IPs for me. No console errors.

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan can you please test yourself in your local machine BEFORE submitting your PRs?

**Comment by Winfidonarleyan** [comment]
> @Winfidonarleyan can you please test yourself in your local machine BEFORE submitting your PRs?

I thought that this module worked and did not check it.
I test today or tomorrow and fix.

**Comment by 55Honey** [comment]
Nevermind, i didn't add the PR from AC. Gonna test again.

**Comment by 55Honey** [comment]
Compiled and tested again with the PR. Works as intended.
I completely missed that this PR is required, please link it in the PR text for blind-acting ones like me :)

**Comment by FrancescoBorzi** [comment]
@Winfidonarleyan good job and thanks @55Honey for testing this

**Pull request merged**